tokens/ITHEUM-df6f26/info.json

```diff
@@ -21,4 +23,4 @@
     "ITHWEGLDF-8fd842",
     "SITHEUM-e05083"
   ]
-}
+}
```

Comment:
optional: empty line at the end of the file

Reply:
Fixed